dev-packages/browser-integration-tests/suites/integrations/Breadcrumbs/dom/clickWithError/test.ts

```diff
@@ -0,0 +1,25 @@
+import { expect } from '@playwright/test';
+import type { Event } from '@sentry/browser';
+
+import { sentryTest } from '../../../../../utils/fixtures';
+import { getFirstSentryEnvelopeRequest } from '../../../../../utils/helpers';
+
+// see: https://github.com/getsentry/sentry-javascript/issues/768
+sentryTest(
+  'should record breadcrumb accessing the target property of an event throws an exception',
```

Comment:
l: not sure but I think theres and "if" missing:

```suggestion
  'should record breadcrumb if accessing the target property of an event throws an exception',
```